charts/cloudzero-agent/templates/cm.yaml

```diff
@@ -217,7 +217,11 @@ data:
     cloud_account_id: {{ .Values.cloudAccountId }}
     region: {{ .Values.region }}
     cluster_name: {{ .Values.clusterName }}
-    host: {{ .Values.host }}
+    {{ if .Values.aggregator.enabled }}
+    destination: 'http://{{ include "cloudzero-agent.aggregator.name" . }}.{{ .Release.Namespace }}.svc.cluster.local/collector'
```

Comment:
one super nit might be that you could pull the content here into a named template to make sure they're always the same. but that is probably making it less readable